metadata-ingestion/src/datahub/ingestion/source/unity/proxy_types.py

```diff
@@ -64,12 +70,12 @@
 class CommonProperty:
     id: str
     name: str
-    type: str
```

Comment:
Thought this got pretty confusing with all the different `type` values floating around, and it was barely used